homeassistant/components/lutron/__init__.py

```diff
@@ -108,9 +131,17 @@ def setup(hass: HomeAssistant, base_config: ConfigType) -> bool:
             hass.data[LUTRON_DEVICES]["binary_sensor"].append(
                 (area.name, area.occupancy_group)
             )
+    await hass.config_entries.async_forward_entry_setups(config_entry, PLATFORMS)
+
+    return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Clean up resources and entities associated with the integration."""
+    if entry.entry_id in hass.data[DOMAIN]:
+        device = hass.data[DOMAIN].pop(entry.entry_id)
+        await device.cleanup()  # Perform cleanup operations
```

Comment:
You're not storing anything in DOMAIN right now. so this code currently does nothing

Reply:
updated to be more consistent with other integrations and remove uselessness